test/src/edit/DistortableImageEditSpec.js

```diff
@@ -15,9 +15,16 @@ describe("L.DistortableImage.Edit", function() {
 		}).addTo(map);
 
 		/* Forces the image to load before any tests are run. */
-		L.DomEvent.on(overlay._image, 'load', function() { done (); });
+		L.DomEvent.on(overlay._image, 'load', function() { 
+			overlay.editing.enable();
```

Comment:
@sashadev-sky I've enabled `editing` as above.

Reply:
yeah I checked myself and that doesn't work! So it is something that your code changed.. because note the spec works before your code is added. so lets debug and try to figure out why?